components/apimgt/org.wso2.carbon.apimgt.rest.api.gateway.v2/src/main/resources/gateway-api.yaml

```diff
@@ -422,13 +400,10 @@ paths:
     # Get API Artifact
     #-----------------------------------------------------
     get:
-      security:
-        - OAuth2Security:
-            - apim:api_publish
-      x-wso2-curl: "curl -k -H \"Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8\" -X GET \"https://127.0.0.1:9443/api/am/gateway/v1/end-points?apiName=PizzaAPI&version=1.0&tenantDomain=registry.com\""
+      x-wso2-curl: "curl -k -H \"Authorization: Basic YWRtaW46YWRtaW4=\" -X GET \"https://127.0.0.1:9443/api/am/gateway/v2/end-points?apiName=PizzaAPI&version=1.0&tenantDomain=registry.com\""
       x-wso2-request: |
-        GET https://127.0.0.1:9443/api/am/gateway/v1/end-points?apiName=PizzaAPI&version=1.0&tenantDomain=registry.com
-        Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8
+        GET https://127.0.0.1:9443/api/am/gateway/v2/end-points?apiName=PizzaAPI&version=1.0&tenantDomain=registry.com
```

Comment:
```suggestion
        GET https://127.0.0.1:9443/api/am/gateway/v2/endpoints?apiName=PizzaAPI&version=1.0&tenantDomain=registry.com
```

Reply:
We need to rename the resource.
Let's update "Get API Artifact" comment as well.